### PR TITLE
Fix settings modal opening when you click to open userlist

### PR DESCRIPTION
**Pull request by KDSBrowne**
fixes issue #5318

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/settings/component.jsx [hunk 1]
```diff
@@ -217,6 +217,7 @@ class Settings extends Component {
         title={intl.formatMessage(intlMessages.SettingsLabel)}
         confirm={{
           callback: () => {
+            this.props.mountModal(null);
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/settings/component.jsx]: The confirm button already has a function that hides the modal when it resolves the callback.
https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/ui/components/modal/fullscreen/component.jsx#L43

File: bigbluebutton-html5/imports/ui/components/settings/submenus/application/component.jsx [hunk 2]
```diff
@@ -191,7 +191,7 @@ class ApplicationMenu extends BaseMenu {
               >
                 {availableLocales && availableLocales.length > 0 ? (
                   <select
-                    defaultValue={this.formatLocale(this.state.settings.locale)}
+                    defaultValue={this.state.settings.locale}
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/settings/submenus/application/component.jsx]: Why do you remove formatLocale? Does this have to do with the issue?

Comment (KDSBrowne) [reply bigbluebutton-html5/imports/ui/components/settings/submenus/application/component.jsx]: When we use this.formatLocale() in the defaultValue prop, The currently activated locale does not display in the select element if the locale has a region.

e.g: English - "en" will display in the select dropdown, but if you switch to "ja_JP" when you re-open the modal, the select wont display the correct language.

Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/settings/submenus/application/component.jsx]: I think you accidentally fixed #5034 :laughing:  :1st_place_medal:

**Pull request merged**